types/index.ts

```diff
@@ -138,12 +87,40 @@ export interface EngineCondition<T extends AudienceDefinitionFilter> {
   rules: EngineConditionRule[];
 }
 
-export interface PingResponse {
-  gdprApplies?: boolean;
+// Audience cache interfaces
+
+export interface MatchedAudience {
+  id: string;
+  matchedAt: number;
+  expiresAt: number;
+  matchedOnCurrentPageView: boolean;
+}
+
+export interface CachedAudienceMetaData {
+  cachedAt: number;
+  audiences: AudienceMetaData[];
 }
 
+export interface AudienceMetaData {
+  id: string;
+  version: number;
+}
+
+// Storage interfaces
+
+export enum StorageKeys {
+  PAGE_VIEWS = 'edkt_page_views',
+  MATCHED_AUDIENCES = 'edkt_matched_audiences',
+  MATCHED_AUDIENCE_IDS = 'edkt_matched_audience_ids',
+  CACHED_AUDIENCES = 'edkt_cached_audiences',
+  CACHED_AUDIENCE_META_DATA = 'edkt_cached_audience_meta_data',
+}
+
+// Network interfaces
```

Comment:
This is not a true network call - since we removed the references to onion lets keep this comment more in line with the rest please.